src/DependencyInjection/BlackSyliusCookieAlertExtension.php

```diff
@@ -16,7 +16,7 @@ final class BlackSyliusCookieAlertExtension extends AbstractResourceExtension
     public function load(array $config, ContainerBuilder $container): void
     {
         $config = $this->processConfiguration($this->getConfiguration([], $container), $config);
-        $loader = new XmlFileLoader($container, new FileLocator(__DIR__ . '/../Resources/config'));
+        $loader = new XmlFileLoader($container, new FileLocator(__DIR__ . '../../config'));
```

Comment:
Cant be XmlFileLoader, your configuration is in php

Reply:
check